### ISSUE TITLE
Unknown directive error by llvm-mc

### ISSUE THREAD

**Issue by RKX1209** [reswitched/libtransistor#8]
When I build by ```make```, following errors are caused by llvm-mc.
**Environment**
   mac OS Sierra 10.12.5
   LLVM version 5.0.0 

```
$ make
mkdir -p build/lib
llvm-mc -arch=aarch64 lib/crt0.nro.S -filetype=obj -o build/lib/crt0.nro.o
lib/crt0.nro.S:1:1: error: unknown directive
.hidden NORELOC__filestart
^
lib/crt0.nro.S:3:1: error: unknown directive
.hidden NORELOC__nro_modhdr
^
lib/crt0.nro.S:5:1: error: unknown directive
.hidden NORELOC_TEXT_START_
^
lib/crt0.nro.S:6:1: error: unknown directive
.hidden NORELOC_TEXT_END_
^
lib/crt0.nro.S:7:1: error: unknown directive
.hidden NORELOC_RODATA_START_
^
lib/crt0.nro.S:8:1: error: unknown directive
.hidden NORELOC_DYNSTR_START_
^
lib/crt0.nro.S:9:1: error: unknown directive
.hidden NORELOC_DYNSYM_START_
^
lib/crt0.nro.S:10:1: error: unknown directive
.hidden NORELOC_RODATA_END_
^
lib/crt0.nro.S:11:1: error: unknown directive
.hidden NORELOC_DATA_START_
^
lib/crt0.nro.S:12:1: error: unknown directive
.hidden NORELOC_DYNAMIC_START_
^
lib/crt0.nro.S:13:1: error: unknown directive
.hidden NORELOC_GOT_START_
^
lib/crt0.nro.S:14:1: error: unknown directive
.hidden NORELOC_GOT_PLT_START_
^
lib/crt0.nro.S:15:1: error: unknown directive
.hidden NORELOC_DATA_END_
^
lib/crt0.nro.S:16:1: error: unknown directive
.hidden NORELOC_BSS_START_
^
lib/crt0.nro.S:17:1: error: unknown directive
.hidden NORELOC_BSS_END_
```

**Comment by misson20000** [comment]
I've seen this before. I think it only happens on mac for some reason. Needs more investigation. I can't reproduce because I don't have a mac ._.

Try a different (newer) version of LLVM if you can? I use `6.0.0svn-r311507` (built from source).

**Comment by misson20000** [comment]
@SomeoneWeird was it you who was having the same issue?

**Comment by SomeoneWeird** [comment]
It was! I tried LLVM 5, 6 and even git HEAD and they all gave me the same thing. It looks like @vgmoose has found a solution though.

**Comment by misson20000** [comment]
closed with #9

**Comment by RKX1209** [comment]
Thx! It works!
